LICENSE

```diff
@@ -0,0 +1,10 @@
+Copyright 2013 Google Inc. All Rights Reserved.
```

Comment:
Not done(?)

I mean copying the real license, like this file
https://github.com/google/kythe/blob/master/LICENSE